src/pages/verify/e-ids/swedish-bankid.mdx

```diff
@@ -60,6 +60,142 @@ For more information on how to configure your device for test and setup test use
 
 _Note_ that, as is also described on the website, using test BankID users does require a reconfiguration of the BankID application. This means it cannot be used for real BankID. So if you are Swedish and already have BankID on your phone, you may want to use a spare phone for testing.
 
+## BankID Phone Authentication
+
+<a href="https://www.bankid.com/en/foretag/telephone-identification" target="_blank">BankID in telephone calls</a> facilitates user authentication during a phone call.
+
+### Overview
+
+The Phone authentication flow is designed for scenarios where a customer is talking with your customer service representative over the phone, and you need to verify that the caller is who they say they are. 
+This is commonly used when a customer contacts their bank or when a call center agent needs to verify the caller's identity.
+
+#### The process looks as follows:
+
+1. The customer service agent enters the customer's personal identification number (SSN) in their system.
```

Comment:
`(or the customer is prompted to input their SSN via an automated PBX system)` could perhaps also make sense.